2주차/1343.py

```diff
@@ -0,0 +1,10 @@
+## 5분
+S = input()
+
+S=S.replace("XXXX","AAAA")
+S=S.replace("XX","BB")
+
+if S.count("X")!=0 :
```

Comment:
```suggestion
if "X" in S :
```
"X"가 S안에 있는지만 확인할때는 count 함수보단 in 연산자 쓰는게 더 직관적이고, 실행속도도 빨라요